main.go

```diff
@@ -98,15 +98,9 @@ func main() {
 		os.Exit(1)
 	}
 
-	pcfg, err := sharedConfig.LoadFrom(configFile)
+	cfg, err := sharedConfig.Config(configFile)
```

Comment:
Why this rename? I won't fight to death over it, but `sharedConfig.LoadFrom` seems to describe what it is doing better than `sharedConfig.Config` IMHO. Since Go doesn't support method overloads, the best name IMHO would be `LoadFromFile` (if we ever want to add other ways of loading the configuration).

Reply:
I agree. I've renamed this in some phase of writing the code that `LoadFrom` did not make sense anymore. But now it does. Renamed back.